src/test/java/com/google/opensesame/MentorTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.opensesame.servlets;
+
+import java.io.IOException;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class PersonTests {
+  private static final String MENTOR_1_NAME = "Mentor 1";
+  private static final String MENTOR_1_GTHUB = "Mentor_1_GitHub_ID";
+  private static final String PROJECT_1 = "Project 1";
```

Comment:
For these variable names I suggest something like `MENTOR_NAME`, `MENTOR_GITHUB_ID`, and `PROJECT_ID` or such. Reasoning: there's no `MENTOR_2_NAME` right now so there's no need to specify `1`. If we did add a second mentor, we could just make an array and then they'd be indexed in the array and we still wouldn't need `1`, `2`, etc.

And it'd be good to add `_ID` to the ID variables so it's more obvious what they represent.

Reply:
Thank you!